### DISCUSSION TITLE
Is it possible to pass props to the hidden <select /> created by the <Select /> component?

### DISCUSSION THREAD

**Discussion by angelod1as** [radix-ui/primitives#1649] [Help] [answered]
I need to add some `data` information to the `<select />` created by the `<Select />` component.

I can pass this prop to the trigger or the content, but not to the actual `<select />`. I tried passing it to `root` as well.

**Comment by benoitgrelard** [comment, marked as answer]
hey @angelod1as,

It isn't currently possible. What do you need to pass?

**Reply by angelod1as** [reply]
I need a `data-valid` field. But I can find a workaround here, no problem. Thanks for the response :)

**Reply by angelod1as** [reply]
(for some reason I cannot edit my last comment: the `data-valid` would be `data-valid={!error}` from the parent form handler
